index.js

```diff
@@ -35,15 +35,25 @@ function addReactSupport(config, options) {
 
 module.exports = (context, options) => {
     options = {
-        targets: ['browser', 'node'], // Can be an array with `browser` and `node` or an object
+        targets: ['browsers', 'node'], // Can be an array with `browsers` and `node` or an object
```

Comment:
Can we say "and/or node" here to clarify the values can be used by themselves.

Reply:
Updated.